### PR TITLE
Feature/more tests

### PR DESCRIPTION
**Pull request by c0deaddict**
Added a browser test for blind transfer. Alice calls Bob, Bob answers and then transfers the call to Charlie.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
You're exporting delay twice here

File: test/browser/shared/bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+    // Wait until the status indicates a registered device.
+    await page.waitFor('.test-status-registered')
+
+    return Object.assign({options: options}, me)
+}
+
+
+module.exports = {
+    brand: brand,
```
Comment (ghost) [inline test/browser/shared/bootstrap.js]: No need to type the key and the value twice when they are the same or do you prefer this format?

File: test/browser/shared/utils.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+
+function delay(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+
+module.exports = {
+    asyncTest: asyncTest,
```
Comment (ghost) [inline test/browser/shared/utils.js]: Shortcut notation?

**Review by ghost — APPROVED** [review]
Nice upgrade to the test suite!

**Comment by ghost** [comment]
This seems to fail:

BRAND=vialer npm run test


```
browser] <alice> I am calling bob.
error TypeError: str is not iterable
    at Object.keypadEntry (/home/spindle/projects/vialer-js/test/browser/shared/utils.js:41:20)
    at test (/home/spindle/projects/vialer-js/test/browser/alice-calls-bob/index.js:28:17)
    at process._tickCallback (internal/process/next_tick.js:68:7)
not ok 3 exception
  ---
    operator: fail
    at: Test.test (/home/spindle/projects/vialer-js/test/browser/shared/utils.js:30:15)
    stack: |-
      Error: exception
          at Test.assert [as _assert] (/home/spindle/projects/vialer-js/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/home/spindle/projects/vialer-js/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/home/spindle/projects/vialer-js/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/home/spindle/projects/vialer-js/node_modules/tape/lib/test.js:76:32)
          at Test.test (/home/spindle/projects/vialer-js/test/browser/shared/utils.js:30:15)
          at process._tickCallback (internal/process/next_tick.js:68:7)
  ...
[11:57:12] '__test-browser' errored after 4.89 s
[11:57:12] Error in plugin "gulp-tape"

```

File: test/browser/shared/utils.js (hunk 2 above)
Comment (c0deaddict) [reply test/browser/shared/utils.js]: I don't know the shortcut notation, what does that look like?

**Comment by c0deaddict** [comment]
Can you retry with the new vialer-jsrc file (it's on keyhub)?

**Pull request merged**